.github/workflows/pr_test_build.yml

```diff
@@ -54,6 +54,7 @@ jobs:
             /opt/android/cake_wallet/cw_haven/ios/External
             /opt/android/cake_wallet/cw_monero/android/.cxx
             /opt/android/cake_wallet/cw_monero/ios/External
+            /opt/android/cake_wallet/cw_monero/ios/Classes/monero_api.cpp
```

Comment:
nope. this will cache this file as well, but we don't need to cache this file since it's already in the git repo.

so Github stores cached data by a key, so that whenever you want to use this cached data from a workflow you just use that key, so I made that key attribute with the hash value for the script files of building monero and haven dependencies, since those dependencies are the one that are not checked in Git and are ignored

so we wanna add to the key value the hash of the `monero_api.cpp` file, since if we modified in this file then we probably modified in something that would need to build Monero dependencies again